python/ray/serve/_private/deployment_state.py

```diff
@@ -2161,7 +2121,7 @@ def _check_and_update_replicas(self):
 
             if len(pending_allocation) > 0:
                 required, available = pending_allocation[0].resource_requirements()
-                message = (
+                msg = (
```

Comment:
Personally I would prefer to spell those out especially we are not under the character limits here.